### ISSUE TITLE
concurrency

### ISSUE THREAD

**Issue by mdreizin** [guigrpa/oao#13]
It would be nice to have ability to run tasks concurrently like `lerna`

https://github.com/lerna/lerna#--concurrency

**Comment by guigrpa** [comment]
Isn't this a duplicate of #10?

**Comment by guigrpa** [comment]
Implemented in v0.5.4. Could you please check it out?

**Comment by mdreizin** [comment]
@guigrpa Many thanks again. Works like a charm :)

Unfortunately, I have faced a couple of issues with console output:

```txt
yarn run v0.20.3
$ oao all "yarn run build" --parallel 
2017-02-23T09:01:53.343Z           storyboard INFO  ┌── ROOT STORY: Node.js 7.5.0 on Darwin 64-bit, SB 3.0.0 [CREATED]
2017-02-23T09:01:53.344Z           storyboard INFO  Log filter: *:DEBUG

<a lot of empty lines>

 1  ROOT STORY: Node.js 7.5.0 on Darwin 64-bit, SB 3.0.0
  2017-02-23T09:01:53.344Z           storyboard INFO  Log filter: *:DEBUG

<a lot of empty lines>

 1  ROOT STORY: Node.js 7.5.0 on Darwin 64-bit, SB 3.0.0
  2017-02-23T09:01:53.344Z           storyboard INFO  Log filter: *:DEBUG

<a lot of empty lines>

 2  Run cmd yarn run build at packages/app

<a lot of empty lines>

 1  ROOT STORY: Node.js 7.5.0 on Darwin 64-bit, SB 3.0.0
  2017-02-23T09:01:53.344Z           storyboard INFO  Log filter: *:DEBUG

<a lot of empty lines>

 2  Run cmd yarn run build at packages/app

<a lot of empty lines>

 3  Run cmd yarn run build at packages/app-config

<a lot of empty lines>

 1  ROOT STORY: Node.js 7.5.0 on Darwin 64-bit, SB 3.0.0
  2017-02-23T09:01:53.344Z           storyboard INFO  Log filter: *:DEBUG

<a lot of empty lines>

 2  Run cmd yarn run build at packages/app

<a lot of empty lines>

 3  Run cmd yarn run build at packages/app-config

<a lot of empty lines>

 4  Run cmd yarn run build at packages/app-di

<a lot of empty lines>

 1  ROOT STORY: Node.js 7.5.0 on Darwin 64-bit, SB 3.0.0
  2017-02-23T09:01:53.344Z           storyboard INFO  Log filter: *:DEBUG

<a lot of empty lines>

 2  Run cmd yarn run build at packages/app

<a lot of empty lines>

 3  Run cmd yarn run build at packages/app-config

<a lot of empty lines>

 4  Run cmd yarn run build at packages/app-di

<a lot of empty lines>

 5  Run cmd yarn run build at packages/app-registry

<a lot of empty lines>

 1  ROOT STORY: Node.js 7.5.0 on Darwin 64-bit, SB 3.0.0
  2017-02-23T09:01:53.344Z           storyboard INFO  Log filter: *:DEBUG

<a lot of empty lines>

 2  Run cmd yarn run build at packages/app
  <ok>
  2017-02-23T09:01:55.124Z                 yarn INFO  Done in 1.35s.
  2017-02-23T09:01:55.142Z                 yarn INFO  
 3  Run cmd yarn run build at packages/app-config
  <ok>
  2017-02-23T09:01:54.597Z                 yarn INFO  Done in 0.80s.
  2017-02-23T09:01:54.618Z                 yarn INFO  


 5  Run cmd yarn run build at packages/app-registry
  <ok>
  2017-02-23T09:01:54.853Z                 yarn INFO  Done in 1.05s.
✨  Done in 2.23s.
```
